src/bmson/pulse.rs

```diff
@@ -2,47 +2,52 @@
 
 use std::collections::BTreeMap;
 
+use num::traits::SaturatingAdd;
```

Comment:
`u64` has `saturating_add` method (but different signature) since 1.47.0. So I think we don't need to use this.